src/c/data.h

```diff
@@ -30,6 +30,8 @@
 #define EX_MQ_CERTFILE "MessageQueue/Optional/CertFile"
 #define EX_MQ_KEYFILE "MessageQueue/Optional/KeyFile"
 #define EX_MQ_SKIPVERIFY "MessageQueue/Optional/SkipCertVerify"
+#define EX_MQ_TOPIC_CMDREQ "MessageQueue/Topics/CommandRequestTopic"
```

Comment:
Doesn't seem to agree with https://github.com/edgexfoundry/edgex-docs/pull/887/files#diff-548bda31e6e1fbf98c1a7f74eb6f8b3c353e4a2ea5038d7354891ee65cd8f257R123
That PR shows MessageQueue/Internal/Topics/CommandRequestTopic or MessageQueue/External/Topics/CommandRequestTopic

Reply:
Hmm, I'm basing this on https://github.com/edgexfoundry/device-sdk-go/blob/main/example/cmd/device-simple/res/configuration.toml
core-command has a different config map than device services?

Reply:
Ah, yes, I misread the other PR - didn't see the distinction between "Common" and "Ch-Command" configuration sections.  Yours does match the Common Configuration in that PR, as it should.